wire-library/wire-tests/src/jvmJsonTest/kotlin/com/squareup/wire/WireJsonTest.kt

```diff
@@ -138,15 +138,16 @@ class WireJsonTest {
     assertThat(jsonLibrary.fromJson("""{"map_int32_Int32":{"1":2}}""", CamelCase::class.java))
         .isEqualTo(CamelCase.Builder().map_int32_Int32(mapOf(1 to 2)).build())
 
-    // Encoding.
+    // Encoding. Note that Kotlin orders fields differently.
     val camel = CamelCase.Builder()
         .nested__message(NestedCamelCase.Builder().one_int32(1).build())
         ._Rep_int32(listOf(1, 2))
         .IDitIt_my_wAy("frank")
         .map_int32_Int32(mapOf(1 to 2))
         .build()
-    assertThat(jsonLibrary.toJson(camel, CamelCase::class.java)).isEqualTo(
-        """{"nestedMessage":{"oneInt32":1},"RepInt32":[1,2],"iDitItMyWAy":"frank","mapInt32Int32":{"1":2}}""")
+    assertThat(jsonLibrary.toJson(camel, CamelCase::class.java)).isIn(
+        """{"nestedMessage":{"oneInt32":1},"RepInt32":[1,2],"iDitItMyWAy":"frank","mapInt32Int32":{"1":2}}""",
+        """{"RepInt32":[1,2],"mapInt32Int32":{"1":2},"nestedMessage":{"oneInt32":1},"iDitItMyWAy":"frank"}""")
```

Comment:
This is why I made `assertJsonEquals`, you can use it to confirm JSON equality without caring about field orders.

`assertJsonEquals(a: String, b: String)`

Reply:
Yeah, that’s much nicer. Fixed.